stdlib/cgi.rb

```diff
@@ -0,0 +1,13 @@
+class CGI
+  module Escape
```

Comment:
why not CGI::Util like in upstream?

https://github.com/ruby/cgi/blob/master/lib/cgi/util.rb

Reply:
Well, i looked here https://docs.ruby-lang.org/en/master/CGI/Escape.html, otherwise no reason, can change it, no problem